scripts/base/frameworks/analyzer/main.zeek

```diff
@@ -126,6 +126,10 @@ export {
 	global disabled_analyzers: set[Analyzer::Tag] = {
 		ANALYZER_TCPSTATS,
 	} &redef;
+
+	## Adds a port to the table of ports stored for the analyzer framework.
```

Comment:
Suggest to expand this a bit explain difference to `register_for_port`.

Reply:
Also, maybe think about more description name for the function. The "to_table" part doesn't convey any semantics on what's the thing that that table is doing.

Reply:
Yah, it's a tough one. How about making `register_for_port` check if an analyzer tag is a session analyzer. It could do all of the table management still, but only call `__register_for_port` if it needs to.

Reply:
I think https://github.com/zeek/zeek/pull/1838/commits/0aafa1779498ce039213b5a655fb8589c2c00ce5 should resolve this.